src/test/java/org/broadinstitute/hellbender/IntelInflaterIntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.hellbender;
+
+import com.intel.gkl.compression.IntelInflater;
+import org.broadinstitute.hellbender.utils.NativeUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Test that it's possible to load libIntelInflater
+ */
+public class IntelInflaterIntegrationTest extends CommandLineProgramTest {
```

Comment:
Since many of the tests test both the inflater and deflater at once, recommend combining the two test classes into a single `IntelInflaterDeflaterIntegrationTest` class.